src/po/Makefile

```diff
@@ -107,74 +107,114 @@ nb.po: no.po
 ja.sjis.po: ja.po
 	@$(MAKE) sjiscorr
 	rm -f $@
-	iconv -f UTF-8 -t CP932 $< | ./sjiscorr > $@
+	if which msgconv > /dev/null; then \
+	  msgconv --width=79 -t CP932 $< | ./sjiscorr > $@; \
+	else \
+	  iconv -f UTF-8 -t CP932 $< | ./sjiscorr > $@; \
+	fi
 
 sjiscorr: sjiscorr.c
 	$(CC) -o sjiscorr sjiscorr.c
 
 ja.euc-jp.po: ja.po
-	iconv -f UTF-8 -t EUC-JP $< | \
-		$(SED) -e 's/charset=[uU][tT][fF]-8/charset=EUC-JP/' \
-			-e 's/# Original translations/# Generated from $<, DO NOT EDIT/' \
-			> $@
+	rm -f $@
+	if which msgconv > /dev/null; then \
+	  msgconv --width=79 -t EUC-JP -o $@ $<; \
+	else \
+	  iconv -f UTF-8 -t EUC-JP $< | \
+	    $(SED) -e 's/charset=[uU][tT][fF]-8/charset=EUC-JP/' > $@; \
+	fi
+	$(SED) -i.bkp -e \
```

Comment:
sed -i is not portable.

Reply:
Yes, I once read that the "-i" switch is an extension implemented by GNU.

But now, as I checked, it is supported in both BSD and macOS. You just have to be sure to specify the extension of the backup file (you can specify an empty one).
https://man.freebsd.org/cgi/man.cgi?sed
https://www.baeldung.com/linux/gnu-bsd-stream-editor
https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux

I checked on FreeBSD and it works.